src/pages/resources/migration/android/migrate-to-3x.md

```diff
@@ -100,6 +100,48 @@ Some of the APIs available in previous major versions of the Mobile SDK for Andr
 
 The `registerExtension` API for each extension that was deprecated in the 2.x version of the mobile SDK has been removed in the 3.x version of the mobile SDK. See the [Update SDK initialization](#update-sdk-initialization) section for more details.
 
+#### MobileCore
+
+| Removed API | Alternative API |
+| :------------- | :-------------- |
+| MobileCore.registerExtension(Class, ExtensionErrorCallback) | [MobileCore.registerExtensions](../../../home/base/mobile-core/api-reference.md#registerextensions) |
+| MobileCore.start(AdobeCallback)| [MobileCore.registerExtensions](../../../home/base/mobile-core/api-reference.md#registerextensions) registers extensions and starts event processing by default |
+| MobileCore.dispatchEvent(Event, ExtensionErrorCallback)| [MobileCore.dispatch(Event)](../../../home/base/mobile-core/api-reference.md#dispatch--dispatchevent) , [MobileCore.dispatchEventWithResponseCallback(Event, long, AdobeCallbackWithError)](../../../home/base/mobile-core/api-reference.md#dispatch--dispatcheventwithresponsecallback) |
+| MobileCore.dispatchEvent(Event, AdobeCallback, ExtensionErrorCallback)| [MobileCore.dispatchEventWithResponseCallback(Event, long, AdobeCallbackWithError)](../../../home/base/mobile-core/api-reference.md#dispatch--dispatcheventwithresponsecallback) |
```

Comment:
Is this correct? I don't remember an API which takes both AdobeCallback and ExtensionErrorCallback.

Reply:
This is the api that we deprecated in 2.x:  https://github.com/adobe/aepsdk-core-android/blob/main-v2/code/core/src/phone/java/com/adobe/marketing/mobile/MobileCore.java#L1203 
```
 public static boolean dispatchEventWithResponseCallback(
            @NonNull final Event event,
            @NonNull final AdobeCallback<Event> responseCallback,
            @Nullable final ExtensionErrorCallback<ExtensionError> errorCallback) {
```